docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,174 @@
+
+On a new service
+''''''''''''''''
+
+Use the `Create a service <https://api.aiven.io/doc/#tag/Service/operation/ServiceCreate>`_ API to create a new service with CCR enabled. When constructing the API request, add the ``user_config`` object to the request body and nest the ``service_to_join_with`` field inside it. As a value of the ``service_to_join_with`` field, set up a name of the service to which you'd like to connect to enable CCR and create a CCR service pair.
+
+.. code-block:: bash
+
+   curl --request POST                                                   \
+      --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service    \
+      --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                 \
+      --header 'content-type: application/json'                          \
+      --data
+         '{
+            "cloud": "string",
+            "plan": "string",
+            "service_name": "new_service_name",
+            "service_type": "cassandra",
+            "user_config": {
+               "cassandra": {
+                  "datacenter": "datacenter_name"
+               },
+               "service_to_join_with": "service_name"
+            }
+         }'
+
+On an existing service
+''''''''''''''''''''''
+
+Use the `Update service configuration <https://api.aiven.io/doc/#tag/Service/operation/ServiceUpdate>`_ API to modify the configuration of your existing service so that it has CCR enabled. When constructing the API request, add the ``user_config`` object to the request body and nest the ``service_to_join_with`` field inside it. As a value of the ``service_to_join_with`` field, set up a name of the service to which you'd like to connect to enable CCR and create a CCR service pair.
+
+.. note::
+
+   To enable CCR on your service, you can connect to services residing only within datacenters that are different from the datacenter used by your service.
+
+.. code-block:: bash
+
+   curl --request PUT                                                                     \
+      --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service/YOUR_SERVICE_NAME   \
+      --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                                  \
+      --header 'content-type: application/json'                                           \
+      --data
+         '{
+            "user_config": {
+               "service_to_join_with":"service_name"
+            }
+         }'
+
+Related reading
+---------------
+
+* :doc:`OpenSearch® cross-cluster replication</docs/products/opensearch/concepts/cross-cluster-replication-opensearch>`
+* :doc:`Set up cross-cluster replication for OpenSearch</docs/products/opensearch/howto/setup-cross-cluster-replication-opensearch>`
+* :doc:`Enabling cross-cluster replication for Apache Kafka® via Terraform</docs/tools/terraform/reference/cookbook/kafka-mirrormaker-recipe>`
+* `Cassandra® documentation <https://cassandra.apache.org/doc/latest/>`_
```

Comment:
IMHO too generic, maybe this would be better:
https://cassandra.apache.org/doc/latest/cassandra/architecture/dynamo.html#multi-master-replication-versioned-data-and-tunable-consistency